napari/_qt/__init__.py

```diff
@@ -29,11 +30,10 @@
 
 # When QT is not the specific version, we raise a warning:
 if StrictVersion(QtCore.__version__) < StrictVersion('5.12.3'):
-    warn_message = f"""
-    napari was tested with QT library `>=5.12.3`.
-    The version installed is {QtCore.__version__}. Please report any issues with this
-    specific QT version at https://github.com/Napari/napari/issues.
-    """
+    warn_message = trans._(
+        "napari was tested with QT library `>=5.12.3`.\nThe version installed is {QtCore.__version__}. Please report any issues with this specific QT version at https://github.com/Napari/napari/issues.",
+        deferred=True,
+    )
```

Comment:
`QtCore.__version__` need to become a parameter.

Reply:
Fixed!